test/extended/util/url/url.go

```diff
@@ -272,7 +342,24 @@ func (ut *Test) ToShell(i int) string {
 		}
 	}
 	lines = append(lines, `rc=0`)
-	cmd := fmt.Sprintf(`curl -X %s %s -s -S -o /tmp/body -D /tmp/headers %q`, ut.Req.Method, strings.Join(headers, " "), ut.Req.URL)
+	post := ""
+	if strings.ToLower(strings.Trim(ut.Req.Method, " ")) == "post" {
+		post = " -H 'Expect:' "
+		if len(ut.PostBodyFile) > 0 {
+			basename := filepath.Base(ut.PostBodyFile)
+			dirname := filepath.Dir(ut.PostBodyFile)
+			lastsubdir := filepath.Base(dirname)
+			err := ut.Oc.AsAdmin().Run("rsync").Args(dirname, ut.PodName+":"+"/tmp", "--strategy=tar").Execute()
+			if err != nil {
+				e2e.Logf("errror on rsync: %#v", err)
```

Comment:
seems like this should fail completely, would there ever be a reason to expect/want the rsync to fail and then proceed w/ the test?  (also extra r in `error`)

Reply:
ah it was a code debug thing ... I found the `e2e.Logf` more readable than the format ginkgo provided, plus there was some verification I was able to do in the interim ... will address

Reply:
you can always leave the log, but i still assume you'd also want to error out at this point, otherwise you're going to get a confusing failure when curl can't find the file, right?

Reply:
Naw at this point I'm good with just the o.Expect.... that is what I pushed